docs/END_USER_DOCUMENT.md

```diff
@@ -316,8 +316,9 @@ In the example above, when calling `addAuthor`, Mocca will not include the list
 For a slightly more complicated example, let's say you wanted to add a book along with the book's author but don't want to include the list
 of books written by the author. You would use the `ignore` property like this:
 
-```java
+``` java
```

Comment:
adding space will take out the formatting I believe

Reply:
It is actually the other way around. The lack of space breaks formatting.